### ISSUE TITLE
NativeBase components cannot be resolved in IDE (PhpStorm/WebStorm)

### ISSUE THREAD

**Issue by lashae** [GeekyAnts/NativeBase#1075]
## react-native, react and native-base version
RN: 0.45.1 React: 16.0.0-alpha.12 NativeBase: 2.2.0

## Expected behaviour
Since NativeBase has its type definitions file within it (`node_modules/native-base/index.d.ts`) my IDE should be able to resolve imported components. All of my other dependencies can be resolved by PhpStorm without any problem, like: React, React-Native, Redux, propTypes etc.

## Actual behaviour
When I use following statement in my IDE (phpStorm)
```
import { connectStyle, Button, Text } from 'native-base';
```
I get: **Cannot resolve symbol** error for all the components imported.

## Steps to reproduce (code snippet or screenshot)
Create a new project on PhpStorm with React, ReactNative and NativeBase. Then add the following import statement to any JS file.
```
import { connectStyle, Button, Text } from 'native-base';
```

## Screenshot of emulator/device
N/A
## Is the bug present in both ios and android or in any one of them?
N/A
## Any other additional info which would help us debug the issue quicker.
N/A

**Comment by m30do** [comment]
+1

**Comment by stone1549** [comment]
+1

**Comment by ratelgogo** [comment]
+1

**Comment by haciSeydaoglu** [comment]
@darkopetrovic Thank you so much. This worked.

**Comment by acilgit** [comment]
It works, thx

**Comment by narek11** [comment]
thanks, works

**Comment by Swahjak** [comment]
Any updates? The solution, as is says, is "temporary". Every time yarn / npm runs the fix is removed from the file.

**Comment by Manugal** [comment]
@darkopetrovic It works! 

This temporary fix might be a PR ;)

**Comment by kk412027247** [comment]
@darkopetrovic thanks a lot.  I fix it finally, that  confuced  me a lot before.

**Comment by IlyaGulya** [comment]
Guys, any progress here?
This is really annoying bug.

**Comment by SupriyaKalghatgi** [comment]
Fixed with NB [2.3.10](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.10)
